src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -59,7 +59,38 @@
  * <pre>
  * &lt;module name="ConstantName"/&gt;
  * </pre>
- *
+ * <p>
+ *   The following configuration is applicable by default:
+ * </p>
+ * <pre>
+ * &lt;module name="ConstantName"&gt;
+ * &lt;property name="format" value="^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$"/&gt;
```

Comment:
Indentation is lost in javadoc that xdoc has. They should be as similar as possible.